app/views/markets/error.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :breadcrumbs do %>
+  <%= render partial: "/markets/dashboard" %>
+<% end %>
```

Comment:
I can get to this error page when making a new market, which means your link back to the dashboard can cause an error if I'm not logged in.